style.css

```diff
@@ -0,0 +1,280 @@
+*{
+	margin: 0;
+	padding: 0;
+}
+.navbar-brand{
+    font-size: 1.6rem !important;
+    font-family: 'Pacifico', cursive;
+}
+#nav-bar{
+    background-color: #fff;
+}
+.navbar-brand span{
+	color: #ea213b;
+}
+.nav-link{
+    margin-right: 1.9rem;
+}
+.nav-link:hover{
+	border-bottom: 2px solid #ea213b;
+}
+.active-link{
+	border-bottom: 2px solid #ea213b;
+}
+#homepage{
+	background-image: linear-gradient(rgba(0,0,0,0.6), rgba(0,0,0,0.6)), url(https://i.pinimg.com/originals/e1/e8/8e/e1e88e69109c22068694d3894e649370.jpg);
+	background-repeat: no-repeat;
+    background-size: cover;
+    background-attachment: fixed;
+	background-position: center;
+	height: 100vh;
```

Comment:
Avoid styling with ids as much as possible

Reply:
Thanks but this code is outdated. I edited my code where I styled with Ids and pushed to github. You will find the updated version in the "commits" tab under pull requests